### PR TITLE
[enterprise-4.16] OSDOCS-7177-RN: add RN entry

### PR DESCRIPTION
**Pull request by jldohmann**
Version(s): 4.16

Issue: https://issues.redhat.com/browse/OSDOCS-7177

Link to docs preview: https://77690--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-16-release-notes.html#ocp-4-16-raid-intel-vroc_release-notes

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by jldohmann** [comment]
also updated a couple of anchors to links

File: release_notes/ocp-4-16-release-notes.adoc [hunk 1]
```diff
@@ -63,7 +63,11 @@ This release adds improvements related to the following components and concepts.
 [id="ocp-4-16-iscsi-support_{context}"]
 ==== Support for iSCSI boot volumes
 
-With this release, you can now install {op-system} to iSCSI boot devices. Multipathing for iSCSI is also supported. For more information, see xref:../installing/installing_bare_metal/installing-bare-metal.adoc#rhcos-install-iscsi-manual[Installing {op-system} manually on an iSCSI boot device] and xref:../installing/installing_bare_metal/installing-bare-metal.adoc#rhcos-install-iscsi-ibft[Installing {op-system} on an iSCSI boot device using iBFT]
+With this release, you can now install {op-system} to iSCSI boot devices. Multipathing for iSCSI is also supported. For more information, see xref:../installing/installing_bare_metal/installing-bare-metal.adoc#rhcos-install-iscsi-manual_installing-bare-metal[Installing {op-system} manually on an iSCSI boot device] and xref:../installing/installing_bare_metal/installing-bare-metal.adoc#rhcos-install-iscsi-ibft_installing-bare-metal[Installing {op-system} on an iSCSI boot device using iBFT]
```
Comment (mburke5678) [inline release_notes/ocp-4-16-release-notes.adoc:66]:
```suggestion
With this release, you can now install {op-system} to iSCSI boot devices. Multipathing for iSCSI is also supported. For more information, see xref:../installing/installing_bare_metal/installing-bare-metal.adoc#rhcos-install-iscsi-manual_installing-bare-metal[Installing {op-system} manually on an iSCSI boot device] and xref:../installing/installing_bare_metal/installing-bare-metal.adoc#rhcos-install-iscsi-ibft_installing-bare-metal[Installing {op-system} on an iSCSI boot device using iBFT].
```

File: release_notes/ocp-4-16-release-notes.adoc [hunk 2]
```diff
@@ -63,7 +63,11 @@ This release adds improvements related to the following components and concepts.
...
+
+[id="ocp-4-16-raid-intel-vroc_{context}"]
+==== Support for RAID storage using Intel(R) Virtual RAID on CPU (VROC)
+With this release, you can now install {op-system} to Intel(R) VROC RAID devices. For more information on configuring RAID to an Intel(R) VROC device, see xref:../installing/install_config/installing-customizing.adoc#installation-special-config-intel-vroc_installing-customizing[Configuring an Intel(R) Virtual RAID on CPU (VROC) data volume].
```
Comment (mburke5678) [inline release_notes/ocp-4-16-release-notes.adoc:70]: _information about_ seems more popular than _information on_.
```suggestion
With this release, you can now install {op-system} to Intel(R) VROC RAID devices. For more information about configuring RAID to an Intel(R) VROC device, see xref:../installing/install_config/installing-customizing.adoc#installation-special-config-intel-vroc_installing-customizing[Configuring an Intel(R) Virtual RAID on CPU (VROC) data volume].
```

**Comment by mburke5678** [comment]
@jldohmann A couple of small nits. LGTM

**Comment by jldohmann** [comment]
@HuijingHei, @mike-nguyen PTAL, this is the RN entry for intel vroc raid support

**Comment by mike-nguyen** [comment]
I'm not sure if its the preview but the link to installing_customizing VROC page just takes me to the customizing page.  Besides the link, everything else LGTM

**Pull request merged**